apps/login/src/ui/SetPasswordForm.tsx

```diff
@@ -66,32 +66,51 @@ export default function SetPasswordForm({
       authRequestId: authRequestId,
       password: values.password,
     }).catch((error: Error) => {
-      setError(error.message ?? "Could not register user");
+      console.error(error);
```

Comment:
Is this printed to the browser console? What's the point in not passing it to setError?

Reply:
I mean the sensitive data is leaked to the dev console like this or am I missing something?`

Reply:
Yes we better remove it. I have revisited all console logs now besides the `navigator.credentials` requests which are not relevant

Reply:
server errors will not leak to a client component in nextjs btw. https://nextjs.org/docs/app/api-reference/file-conventions/error#errormessage